PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Hey there! So you want to contribute to FirebaseUI? Before you file this pull request, follow these steps:
+
+  * Have you read [the contribution guidelines](CONTRIBUTING.md)
```

Comment:
"Have you read [the contribution guidelines](CONTRIBUTING.md)?" (add punctuation)

Or, since it says "follow these steps" make it imperative: "Read [the contribution guidelines](CONTRIBUTING.md)."